_posts/2024-05-20-highlights-otp-27.md

```diff
@@ -360,14 +360,29 @@ After `B` follows the start delimiter, in this case `[`.  Since no escape charac
 are allowed, it is necessary to choose delimiters that don't occur in the string
 contents. After the contents follows the end delimiter, in this case `]`.
 
-The `B` sigil is the default sigil that is used if the letter following
-`~` is omitted. Thus we get the same binary and the same output if we omit the `B`:
+`~b` creates a binary in the same way as `~B`, except that backslashes
+will be interpreted as escape characters. This can be useful if one
+wants to insert control characters such as TAB (`\t`) into a binary:
 
 ```
-greek_quote() ->
-    S = ~["Know thyself" (Greek: Γνῶθι σαυτόν)],
-    io:format("~ts\n", [S]).
+1> ~b"abc\txyz".
+<<"abc\txyz">>
+```
+
+Here we used the `"` character as delimiters as it is not used within
+the string.
+
+If we omit the letter after `~`, we will get the same result:
+
 ```
+2> ~"abc\txyz".
+<<"abc\txyz">>
+```
+
+The default sigil (no letter following `~`) forces a binary to be
```

Comment:
Förslag:
The default sigil (no letter following `~`) creates a binary, just like `~b` and `~B`, but whether or not escape characters are interpreted depends on the form of the string. As mentioned above, triple-quoted strings do not by default interpret escape sequences such as `\n`, but plain inline strings do, so `~"abc\ndef"` works as you might expect, and you can always prefix an existing string like `"abc\ndef"` with a `~` to turn it into a binary without fear of changing its content.